### PR TITLE
Add Thinkful Data Science

### PR DESCRIPTION
**Pull request by AaronFried**

### CODE REVIEW

File: bootcamps/thinkful/programs/flexible-data-science-bootcamp/description.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Thinkful's data science bootcamp is driven by 1-on-1 mentorship. By working closely with a mentor, students can learn the topics they need to get a job. That includes Python, its math libraries such as NumPy and Matplotlib, and mathematical modeling using linear algebra and statistics. By using these industry tools and knowledge sets, Thinkful students can get ahead with a mentor-personalized experience. Students meet with their mentor three times a week for an hour each session, providing students with best practices and constant code review.
```
Comment (Grae-Drake) [inline bootcamps/thinkful/programs/flexible-data-science-bootcamp/description.md]: Suggested revision:

```
Thinkful's data science bootcamp is driven by 1-on-1 mentorship. Students learn the skills they need to get a job by working closely with a professional data scientist. That includes programming in Python, related Python packages for analysis, modeling, and machine learning like NumPy, SciPy, pandas, and scikit-learn; plus data visualization tools like matplotlib, seaborn, and Jupyter. Thinkful students get ahead with a mentor-personalized experience, customizing their learning path and focusing on the skills they need. Students meet with their mentor three times a week for an hour each session to learn best practices, receive constant feedback, and move quickly.
```

File: bootcamps/thinkful/programs/flexible-data-science-bootcamp/data.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+
+admissions: Open to anyone
+cities:
+- remote
+- atlanta
+- raleigh
+- raleigh-durham
```
Comment (learykara) [inline bootcamps/thinkful/programs/flexible-data-science-bootcamp/data.yml]: should this be DC now?

File: bootcamps/thinkful/programs/flexible-data-science-bootcamp/data.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+commitment: part-time
+cost_description: $1,500 per month
+display_name: Flexible Data Science Bootcamp
+duration: 6
+duration_units: months
+financing: 'Yes'
+guarantee: No
+placement: As a brand-new program, Thinkful aims to carry over its 93% success rate in job placement from Web Development Bootcamp.
+program_slug: flexible-data-science-bootcamp
+promises_job: false
```
Comment (learykara) [inline bootcamps/thinkful/programs/flexible-data-science-bootcamp/data.yml]: really?

File: bootcamps/thinkful/programs/flexible-data-science-bootcamp/data.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+- atlanta
+- raleigh
+- raleigh-durham
+commitment: part-time
+cost_description: $1,500 per month
+display_name: Flexible Data Science Bootcamp
+duration: 6
+duration_units: months
+financing: 'Yes'
+guarantee: No
```
Comment (learykara) [inline bootcamps/thinkful/programs/flexible-data-science-bootcamp/data.yml]: no guarantee?

File: bootcamps/thinkful/programs/flexible-data-science-bootcamp/data.yml (hunk 2 above)
Comment (AaronFried) [inline bootcamps/thinkful/programs/flexible-data-science-bootcamp/data.yml]: adding!

File: bootcamps/thinkful/programs/flexible-data-science-bootcamp/data.yml (hunk 3 above)
Comment (AaronFried) [inline bootcamps/thinkful/programs/flexible-data-science-bootcamp/data.yml]: pinged DS and GD to verify...

File: bootcamps/thinkful/programs/flexible-data-science-bootcamp/data.yml [hunk 5]
```diff
@@ -0,0 +1,30 @@
+admissions: Open to anyone
```
Comment (tholex) [inline bootcamps/thinkful/programs/flexible-data-science-bootcamp/data.yml]: I believe there's an admissions process as well... not sure if "Open Admissions" is valid. CC/ @Grae-Drake

Comment (AaronFried) [inline bootcamps/thinkful/programs/flexible-data-science-bootcamp/data.yml]: 'Must complete prep course' ?

**Pull request merged**